<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>671</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>671</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtVB</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images.png</normaloff>images.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>401</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>dsds</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabWidgetPage1">
     <attribute name="title">
      <string>Accueil</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Visu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_visu">
          <property name="maximum">
           <number>16</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Autotest</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_autotest">
          <property name="maximum">
           <number>16</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>2</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="LSV">
              <property name="text">
               <string>LS-V</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="LSFU">
              <property name="text">
               <string>LS-FU</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="Panne_sol">
              <property name="text">
               <string>Sol</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="Panne_engin">
              <property name="text">
               <string>Engin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>15</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="LSSF">
              <property name="text">
               <string>LSSF</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="connectBtn">
              <property name="text">
               <string>Générer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="generateCopyBtn">
              <property name="text">
               <string>Générer et copier</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="connectSerialBtn">
              <property name="text">
               <string>Connect serial</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="disconnectSerialBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Disconnect serial</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="genSendSerialBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Gen et envoyer en série</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="VAL">
          <property name="text">
           <string>VAL</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="MV">
          <property name="text">
           <string>MV</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="FC">
          <property name="text">
           <string>FC</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Générateur NG</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>591</width>
        <height>231</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-image: url(:/img/KVB_haut.png);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="FCbtn">
       <property name="geometry">
        <rect>
         <x>227</x>
         <y>160</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="KVB.qrc">
         <normaloff>:/img/FC_OFF.png</normaloff>:/img/FC_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="MVbtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>48</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="KVB.qrc">
         <normaloff>:/img/MV_OFF.png</normaloff>:/img/MV_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="VALbtn">
       <property name="geometry">
        <rect>
         <x>98</x>
         <y>160</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="KVB.qrc">
         <normaloff>:/img/VAL_OFF.png</normaloff>:/img/VAL_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="LSSFbtn">
       <property name="geometry">
        <rect>
         <x>436</x>
         <y>31</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/LSSF_OFF.png</normaloff>img/LSSF_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>76</width>
         <height>76</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="LSVBtn">
       <property name="geometry">
        <rect>
         <x>346</x>
         <y>59</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="KVB.qrc">
         <normaloff>:/img/Lampe_OFF.png</normaloff>:/img/Lampe_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>76</width>
         <height>76</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="LSFUBtn">
       <property name="geometry">
        <rect>
         <x>389</x>
         <y>58</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="KVB.qrc">
         <normaloff>:/img/Lampe_OFF.png</normaloff>:/img/Lampe_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>76</width>
         <height>76</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="PanneSolBtn">
       <property name="geometry">
        <rect>
         <x>357</x>
         <y>173</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="KVB.qrc">
         <normaloff>:/img/Lampe_OFF.png</normaloff>:/img/Lampe_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>76</width>
         <height>76</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="PanneEnginBtn">
       <property name="geometry">
        <rect>
         <x>401</x>
         <y>172</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="KVB.qrc">
         <normaloff>:/img/Lampe_OFF.png</normaloff>:/img/Lampe_OFF.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>76</width>
         <height>76</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>253</y>
        <width>591</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QTextEdit" name="JSONTextEditNG"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="GenNGBtn">
           <property name="text">
            <string>Générer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="GenCopyNGBtn">
           <property name="text">
            <string>Générer et copier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="GenSerialNGBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Gen et envoyer en série</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Paramètres série</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>189</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Port série</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serialPortName"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="KVB.qrc"/>
 </resources>
 <connections/>
</ui>
